frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -10,7 +10,7 @@ import {
   StackDivider,
   Text,
 } from '@chakra-ui/react'
-import moment from 'moment'
```

Comment:
suspecting moment is causing chromatic to fail - replacing with FE library

Reply:
The reason was because `moment` didn't exist?

Reply:
yes - moment is in the overall package.json but not the frontend's package.json. changing to re-use `fns-date` fixes it as it is in the fe's package.json